packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/add-orgs.test.ts

```diff
@@ -0,0 +1,105 @@
+import test, { Test } from "tape-promise/tape";
+
+import {
+  FabricTestLedgerV1,
+  pruneDockerAllIfGithubAction,
+} from "@hyperledger/cactus-test-tooling";
+
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+import path from "path";
+import { STATE_DATABASE } from "@hyperledger/cactus-test-tooling";
+import { LedgerStartOptions } from "../../../../../../cactus-test-tooling/dist/types/main/typescript/fabric/fabric-test-ledger-v1";
```

Comment:
@RafaelAPB Need to import via the official package name otherwise this will be broken in production deployments, so instead of the relative path you need to import ` from "@hyperledger/cactus-test-tooling";`

And if it doesn't compile that way it must be because you forgot to `export` in the `public-api.ts` file of the test tooling package.